.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ version: 2
 # Run static code analyzer for SourceCloud
 # Upload test results
 # Every run command should start with source ${INIT_ENV} to ensure all default dependancies are available
-
+# TODO : Move this to github actions to take advantage of dynamic base branch selection
```

Comment:
NIT: We should probably create github or JIRA ticket (in devops backlog) to properly track this

Reply:
https://usdot-carma.atlassian.net/browse/CAR-6020